### PR TITLE
fix: sentry no longer supports tracing as a seperate package

### PR DESCRIPTION
**Pull request by ageddesi**
I recently installed the sentry experimental setup and noticed that it is using deprecated implementations. I have updated these implementations to now use what they are expecting from the latest update. 

Let me know if you think I have missed anything.

### CODE REVIEW

**Comment by ahaywood** [comment]
Awesome! Thanks for submitting! We'll take a look.

File: packages/cli/src/commands/experimental/templates/sentryWeb.ts.template [hunk 1]
```diff
@@ -1,5 +1,5 @@
+import { BrowserTracing } from '@sentry/browser'
```
Comment (Tobbe) [inline packages/cli/src/commands/experimental/templates/sentryWeb.ts.template:1]: I think we can get rid of this, and instead just do `Sentry.BrowserTracing`, using `Sentry` from the `'@sentry/react'` import below. It's what they do here https://docs.sentry.io/platforms/javascript/guides/react/#configure

How does that sound to you @ageddesi?

Comment (ageddesi) [reply packages/cli/src/commands/experimental/templates/sentryWeb.ts.template:1]: Yer sounds good, looks like I left that in by mistake :P

**Review by Josh-Walker-GM — APPROVED** [review]
Looks good, follows the sentry migration guide and I tested it locally where it worked well.

I am sorry it took me this long to get to reviewing this. Thanks for the PR it's hugely appreciated!

**Pull request merged**